homework/tests/example.spec.js

```diff
@@ -27,5 +27,130 @@ data.forEach(version => {
   
       await expect(page.locator('#numberAnswerField')).toHaveValue('5');
     });
+
+    test('Multiplying 2.5 and -3 results in -7', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
URL is repeated in every single test. 
In case of url change -> there will be plenty of placesin the test file to change it. Please declare a variable for this.